### DISCUSSION TITLE
is this project abandoned?

### DISCUSSION THREAD

**Discussion by fahadshery** [lllyasviel/Fooocus#3613] [General]
Hi,

there are number of PR's opened and the last updates were few months ago now?
I was wondering if this project is still live?

Cheers

**Comment by ILikeToasters** [comment]
Not really abandoned. https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#project-status-limited-long-term-support-lts-with-bug-fixes-only

**Comment by p1agnut** [comment]
I wish I had the necessary python skills to contribute something valuable to the project. 😢

**Comment by AriesMillion** [comment]
Hi, 
I have now spent almost 2 days on installing Focuuus, but still unsuccessfully. I have also paid for online version with one year subscriptions and today it stopped working. I have now tried on Colab, but I see there are limited resources. 
Is anyone here available for help with install Fooocus. 

I have tried at first with wrong version, 2.1 and latest Python, which I later discovered that latest version is 2.5.5. and should run Python between 3.8 - 3.11.x

Any help would be appreciated, I can even compensate for time if you make me local install.

Regards

**Reply by maryf-t-s** [reply]
https://www.youtube.com/watch?v=j1WuQndmgFE&list=PLSofK8pvSk5oHH9iNAAemQREADEY7CPp_  
pay special attention to the graphics card requirements

**Comment by muratyuceer** [comment]
why LTS? Keep working

**Reply by khaldi-yass** [reply]
Is that an order or a threat sir?

**Reply by mashb1t** [reply]
@muratyuceer feel free to continue development of whichever features you like to add by forking the repo. Happy to do QA, just shoot me a message.

**Comment by FabioLe23** [comment]
acly krita ai diffusion
https://github.com/Acly/krita-ai-diffusion
